sb.web.app/src/components/Navbar/Notifications.vue

```diff
@@ -33,7 +33,7 @@
                     <router-link :to="{name:'Profile', params:{tab: 'purchases'}}" @click.prevent="">
                         <div id="new-list-content">
                         <img class="notice-img" src="../../assets/navbar_logos/notice.png" alt="{{ $t('notifications.new_notification') }}"/>
-                        <p class="notice-desc">{{ item.fromUser }} har godkänt din köpförfrågan{{ $t('notifications.go_to') }} <u> {{ $t('nav.my_account') }} </u> {{ $t('notifications.download_invoice') }}</p>
+                        <p class="notice-desc">{{ item.fromUser }} has approved your purchase request{{ $t('notifications.go_to') }} <u> {{ $t('nav.my_account') }} </u> {{ $t('notifications.download_invoice') }}</p>
```

Comment:
Sorry but this needs to be done via i18n - we need to be able to find *all* the English that needs to be transalted back to Swedish later .. if we do it like this we're gonna have the reverse problem of having to find allt he English to remove later on. 


Take a look at some of the mor examples here https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting for how to handle this in a way that is not English specific